app/models/workflow_step.rb

```diff
@@ -7,6 +7,10 @@ class WorkflowStep < ApplicationRecord
   validates :process, presence: true
   validates :version, presence: true
   validates :repository, presence: true
+  validates :status, presence: true
+  validate :valid_status?
+
+  VALID_STATUS = %w[waiting completed queued error skipped].freeze
```

Comment:
do we need `waiting`?  Last time I checked we don't use this.

Reply:
i think so -- isn't that the default state for new workflow steps?

Reply:
Sorry, that's right. I was thinking of "hold".